lib/locales/es.js

```diff
@@ -11,7 +11,64 @@
+        'Click to enter X axis title': 'Haga clic para introducir el título del eje X', // plots/plots.js:301
+        'Click to enter Y axis title': 'Haga clic para introducir el título del eje Y', // plots/plots.js:302
+        'Compare data on hover': 'Comparar datos al pasar por encima', // components/modebar/buttons.js:167
+        'Double-click on legend to isolate one trace': 'Haga doble-clic en la leyenda para aislar una traza', // components/legend/handle_click.js:90
+        'Double-click to zoom back out': 'Haga doble-clic para restaurar la escala', // plots/cartesian/dragbox.js:335
+        'Download plot as a png': 'Descargar gráfica como png',       // components/modebar/buttons.js:52
+        'Edit in Chart Studio': 'Editar en Chart Studio',             // components/modebar/buttons.js:76
+        'IE only supports svg.  Changing format to svg.': 'IE solo soporta svg. Cambiendo formato a svg.', // components/modebar/buttons.js:60
+        'Lasso Select': 'Seleccionar con lazo',                       // components/modebar/buttons.js:112
+        'Orbital rotation': 'Rotación esférica',                      // components/modebar/buttons.js:279
+        'Pan': 'Modo Panorámica',                                     // components/modebar/buttons.js:94
+        'Produced with Plotly': 'Hecho con Plotly',                   // components/modebar/modebar.js:256
+        'Reset': 'Reiniciar',                                         // components/modebar/buttons.js:431
+        'Reset axes': 'Reiniciar ejes',                               // components/modebar/buttons.js:148
+        'Reset camera to default': 'Restaurar cámara predeterminada', // components/modebar/buttons.js:313
+        'Reset camera to last save': 'Restaurar anterior cámara',     // components/modebar/buttons.js:321
+        'Reset view': 'Restaurar vista',                              // components/modebar/buttons.js:582
+        'Reset views': 'Restaurar vistas',                            // components/modebar/buttons.js:528
+        'Show closest data on hover': 'Mostrar el dato más cercano al pasar por encima', // components/modebar/buttons.js:157
+        'Snapshot succeeded': 'La captura de la instantánea finalizó correctamente', // components/modebar/buttons.js:66
+        'Sorry, there was a problem downloading your snapshot!': '¡La descarga de la instantánea falló!', // components/modebar/buttons.js:69
+        'Taking snapshot - this may take a few seconds': 'Capturando una instantánea - podría tardar unos segundos', // components/modebar/buttons.js:57
+        'Toggle Spike Lines': 'Mostrar/Ocultar Guías',                // components/modebar/buttons.js:547
+        'Toggle show closest data on hover': 'Activar/Desactivar mostrar el dato más cercano al pasar por encima', // components/modebar/buttons.js:352
+        'Turntable rotation': 'Rotación plana',                       // components/modebar/buttons.js:288
```

Comment:
Oh that's cool - it's like planar vs spherical rotation? That may be a good reference for other translators in the future 🏆

Reply:
For this translation, I followed [Blender](https://svn.blender.org/svnroot/bf-translations/branches/es/es.po).